test/mock/nock.js

```diff
@@ -0,0 +1,54 @@
+/* jshint esversion:6 */
+/* global __dirname */
+
+(function(){
+  'use strict';
+  
+  const nock = require('nock');
+  const Promise = require('bluebird');
+  
+  class NockController {
+    static nockSettings(url, method, route) {
+      const routeParts = route.split('/');
+      let fileName = '';
+      
+      for (let i = 0; i < routeParts.length; i++) {
+        if (routeParts[i].length > 0) {
+          fileName += routeParts[i] + '-';
+        } 
+      }
+      
+      fileName = fileName.substring(0, fileName.length - 1);
```

Comment:
it would be cleaner to construct this with array.join (push parts into array and join them with '-' -separator)